docs/authentication/configuration/restrictions.mdx

```diff
@@ -1,54 +1,76 @@
 ---
-sanity_slug: /docs/authentication/restrictions
+title: Restrictions | Authentication
+description: Learn how to manage user access to your application by configuring Clerk's access restrictions.
 ---
 
-# Overview
+# Restrictions
 
-# Allowlist
+Clerk provides a set of restriction options designed to provide you with enhanced control over who can gain access to your application. Through these restriction settings, you can limit sign-ups or prevent accounts with specific identifiers from accessing your application.  These identifiers can be email addresses, phone numbers, and even entire domains. To get started, navigate to the [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions) tab under the **User & Authentication** section of the [Clerk Dashboard](https://dashboard.clerk.com).
```

Comment:
There are 2 spaces before `These identifiers ...`

Reply:
You don't need a comma `,` before `and` : `phone numbers, and ...`

Reply:
> You don't need a comma `,` before `and` : `phone numbers, and ...`

actually, you do; its called an oxford comma!

Reply:
:nerd_face: TIL